test/functional/VersionedLayerClient.test.ts

```diff
@@ -0,0 +1,94 @@
+/*
```

Comment:
Path should be `<root>/tests/functional/....` not `test/`

Reply:
updated